src/Pages/LikedVideosPage/LikedVideosPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+import { getLocalStorage } from '../../Helpers/Helpers/Utils';
+import axios from 'axios';
+import { VideoCard } from '../../Shared';
+
+const LikedVideosPage = () =>{
+
+    const auth = getLocalStorage("authData");
+    const [ likedVideosData, setLikedVideosData] = useState([])
+
+    const getlikedVideos = async () => {
+        try {
+          const response = await axios.get('/user/likes', { headers: { authorization: auth } });
+          if(response.status === 200){
+              setLikedVideosData(response?.data?.likes);
+          }
+        } catch (error) {
+          console.error(error);
```

Comment:
Suggestion: You can use alert for showing the error

Reply:
ty! will do that